README.md

```diff
@@ -15,12 +15,14 @@ GitHub: [@JUPAC](https://silver-shortbread-797321.netlify.app/)
 - CSS.
 - JS.
 - React.
+- Redux
+- Tailwind
 - Gitflow.
 - Linters.
 
 To get a local copy up and running follow these simple steps:
 
-1. Go to the [repository page]( https://juaryimami.github.io/math-magician/).
+1. Go to the [repository page]( https://juaryimami.github.io/book-store-react/).
 2. Press the "Code" button and copy the link.
 3. Clone it using git command `git clone <link>`.
 
```

Comment:
- [ ] Kindly update your information here on how to set up your app using `craco` config.

- [ ] Also remember to change `math-magician` here to the correct repo name

  ![hff](https://user-images.githubusercontent.com/54848363/201165722-11d0daeb-c84f-4fc4-b63f-7a371f8f3046.jpg)